deploy/chart/catalog_resources/rh-operators/descheduler.csv.yaml

```diff
@@ -0,0 +1,142 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    categories: openshift optional
+    certifiedLevel: Primed
+    containerImage: registry.svc.ci.openshift.org/openshift/origin-v4.0:descheduler-operator
+    createdAt: 2019/11/15
+    description: An operator to run the OpenShift descheduler
+    healthIndex: B
+    longDescription: |
+      # Descheduler for Kubernetes
```

Comment:
Why is this an annotation and not in `spec.description`?

Reply:
Moved to `spec.description`